source/install/config-ssl-http2-nginx.rst

```diff
@@ -263,7 +304,49 @@ You may need to update the Callback URLs for the Application entry of Mattermost
 
 1. Log in to your GitLab instance as the admin.
 2. Go to **Admin > Applications**.
-3. Click **Edit** on GitLab-Mattermost.
+3. Select **Edit** on GitLab-Mattermost.
 4. Update the Callback URLs to your new domain/URL.
```

Comment:
```suggestion
4. Update the callback URLs to your new domain/URL.
```

Reply:
@justinegeffen it looked like this was suggested twice. See the screenshot. Other than that, I ran back through the suggestions and it looked good!

![Screen Shot 2021-02-09 at 11 21 31 AM](https://user-images.githubusercontent.com/46071821/107393862-3ed43f00-6ac9-11eb-8e95-cc0ea480e56a.png)